rtgui/histogrampanel.cc

```diff
@@ -869,105 +869,72 @@ void HistogramArea::updateBackBuffer ()
         LUTu& gh = rawMode ? ghistRaw : ghist;
         LUTu& bh = rawMode ? bhistRaw : bhist;
 
-        // make double copies of LUT, one for faster access, another one to scale down the raw histos
```

Comment:
@heckflosse I didn't see the need for these additional arrays, but maybe they are really useful in getting a speedup? At least the intended 'scaling' is no longer necessary because of my alternative histogram scaling.